### DISCUSSION TITLE
Unit not functioning after test

### DISCUSSION THREAD

**Discussion by pmgducati** [darthcloud/BlueRetro#268] [Q&A] [answered]
Hi @darthcloud,
I created some PCBs for someone and everything worked perfectly for me.  I made a PCE and Saturn AIO set and tested with 2x 8bitdo SN30 Pro+ controllers.  Everything tested great on both systems over many games for 2+ hours per system.  Once the person received the units, and got a controller paired it is not working properly, they are only able to get a couple inputs (X and Up I believe.)

So here is my question:  Would my controllers connecting first create some sort of profile that is messing with the controllers that the owner is trying to use?  

Should the unit be reflashed?

I sent the link to the discussion to the user so he can give some input too.

**Comment by darthcloud** [comment, marked as answer]
Nothing need to be reset or reflash.

Which controller they use? Xbox controller need to use latest FW from MS to work properly as noted here:
https://github.com/darthcloud/BlueRetro/wiki/Controller-pairing-guide#xbox-one-s-series-xs--adaptive-controller

**Reply by pmgducati** [reply]
He got it figured out, turns out it was the controller and not the unti itself.  Thanks for all the help though!
